tests/e2e/specs/backend/product-query/atomic-blocks.test.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import { canvas, setPostContent, insertBlock } from '@wordpress/e2e-test-utils';
+import { visitBlockPage, saveOrPublish } from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { GUTENBERG_EDITOR_CONTEXT, describeOrSkip } from '../../../utils';
+import { shopper } from '../../../../utils';
```

Comment:
In the previous meetup, we have decided to try to move away from relative paths as much as we can. Specifically, no ancenstor relative paths, because they can be hard to read and also hard to maintain. More on the reasoning in this [P2 post](https://teamkirigami.wordpress.com/2022/03/24/meetup-code-standards-and-practices-brainstorm/#relative-paths-vs-absolute-paths). 

Maybe you can create an alias in the `tsconfig` that points to the base test directory, and import from there?

Reply:
We have three places from importing utilities in our test at the moment:
- `tests/utils/`.
- `tests/e2e/utils.js`.
- `tests/e2e/page-utils.js`

`@woocommerce/blocks-test-utils` is actually an alias that points to `tests/utils` which contains utilities for E2E and Unit testing. We have discussed organizing test utilities [here](https://github.com/woocommerce/woocommerce-blocks/issues/5893) and [here](https://github.com/woocommerce/woocommerce-blocks/issues/6205). I think it's good to start working on https://github.com/woocommerce/woocommerce-blocks/issues/6205 after this PR. And I agree with @nielslange suggestion [here](https://github.com/woocommerce/woocommerce-blocks/issues/5893#issuecomment-1046829849):

> My thoughts are similar to @sunyatasattva's thoughts. I'd rather move all functionality to test/utils so that these functions can be used both by unit, integration and e2e tests.

Reply:
Alright so you are saying: let's keep the relative paths here, and then address the organization in another PR, and in that PR fix those relative paths to absolutes?